docs/output.md

```diff
@@ -392,6 +392,21 @@ By default, only the raw bins (and unbinned contigs) from the actual binning met
 
 ⚠️ Due to ability to perform downstream QC of both raw and refined bins in parallel (via `--postbinning_input)`, bin names in DAS Tools's `*_allBins.eval` file will include `Refined`. However for this particular file, they _actually_ refer to the 'raw' input bins. The pipeline renames the input files prior to running DASTool to ensure they can be disambiguated from the original bin files in the downstream QC steps.
 
+### Tiara
+
+Tiara is a contig classifier that identifies the domain (prokarya, eukarya) of contigs within a file. This is used in this pipeline to find the most likely domain classification of each bin or unbin based on its contig identities.
```

Comment:
I wonder if it to make it clear it's in the same scope as DAS_Tool by explicitly writing that it 'refines' bins along classification terms?

I also wonder if we should now have different subdirectories now to also guide users that not everything in  GenomeBinning is the same thing? E.g.

```
GenomeBinning/
    Binning/
        MaxBin
        MetaBat
        Concoct
    BinRefinement/
        DASTool
        Tiara
```

Or something - thoughts (also at @d4straub @skrakau)?

Reply:
> Alternatively, Tiara might go in the Taxonomy directory as it's technically broadly classifying bins taxonomically?

This sounds good to me! Then it would be together in a folder with centrifuge, kraken2, CAT and GTDBTK taxonomy assignments (the latter 2 on bins, the former 2 on reads).

Reply:
OK fair enough! Then yes, a separate directory works :)